playbooks/common/openshift-cluster/config.yml

```diff
@@ -34,3 +34,5 @@
 - include: additional_config.yml
 
 - include: ../openshift-node/config.yml
+
+- include: openshift_router.yml
```

Comment:
Should this playbook be called openshift_hosted instead?

I'm also wondering if we should take this opportunity to rename the openshift_router role to openshift_hosted to start the consolidation of the "hosted" roles into a openshift_hosted role...